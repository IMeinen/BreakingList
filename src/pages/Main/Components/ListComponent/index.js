import React from 'react';

import PropTypes from 'prop-types';
import { ItemsList, CharItem } from './styles';

function ListComponent({
  currentCharacters,
  setOpenModal,
  setSelectedChar,
  characters
}) {
  const handleOpenModal = (id) => {
    setOpenModal(true);
    const selected = characters.filter((item) => {
      return item.char_id === id;
    });
    setSelectedChar(selected);
  };

  return (
    <ItemsList>
      {currentCharacters &&
        currentCharacters.map((char) => {
          return (
            <CharItem
              onClick={() => handleOpenModal(char.char_id)}
              test-id={`List item of ${char.char_id}`}
              key={`List item of ${char.char_id}`}
            >
              <img
                src={char.img}
                alt={`img of ${char.char_id}`}
                height="100%"
                width="100%"
              />
              <div className="description">
                <h1>{char.name}</h1>
              </div>
            </CharItem>
          );
        })}
    </ItemsList>
  );
}

export default ListComponent;
ListComponent.propTypes = {
  characters: PropTypes.arrayOf(
    PropTypes.shape({
      appearance: PropTypes.arrayOf(PropTypes.number),
      better_call_saul_appearance: PropTypes.arrayOf(PropTypes.number),
      birthday: PropTypes.string,
      category: PropTypes.string,
      char_id: PropTypes.number,
      img: PropTypes.string,
      name: PropTypes.string,
      nickname: PropTypes.string,
      occupation: PropTypes.arrayOf(PropTypes.string),
      portrayed: PropTypes.string,
      status: PropTypes.string,
    })
  ).isRequired,
  currentCharacters: PropTypes.arrayOf(
    PropTypes.shape({
      appearance: PropTypes.arrayOf(PropTypes.number),
      better_call_saul_appearance: PropTypes.arrayOf(PropTypes.number),
      birthday: PropTypes.string,
      category: PropTypes.string,
      char_id: PropTypes.number,
      img: PropTypes.string,
      name: PropTypes.string,
      nickname: PropTypes.string,
      occupation: PropTypes.arrayOf(PropTypes.string),
      portrayed: PropTypes.string,
      status: PropTypes.string,
    })
  ).isRequired,
  setOpenModal: PropTypes.func.isRequired,
  setSelectedChar: PropTypes.func.isRequired,
};
