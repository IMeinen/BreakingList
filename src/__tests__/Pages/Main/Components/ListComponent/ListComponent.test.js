import React from 'react';

import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import fetchData from '../../../../../__mocks__/Api'
import ListComponent from '../../../../../pages/Main/Components/ListComponent';

Enzyme.configure({ adapter: new Adapter() });
describe('ListComponent tests', () => {
  it('renders correctly', async () => {
    const mocked = await fetchData();

    if (mocked !== undefined) {
      const ComponentProps =  {
        currentCharacters: mocked,
        setOpenModal: console.log,
        setSelectedChar: console.log,
        characters: mocked,
      };

      const wrapper = shallow(<ListComponent {...ComponentProps} />);

      expect(wrapper).toMatchSnapshot();
    }
  });
});
