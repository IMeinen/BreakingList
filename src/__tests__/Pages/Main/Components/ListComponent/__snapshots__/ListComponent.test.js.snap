// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListComponent tests renders correctly 1`] = `
<styled.ul>
  <styled.li
    key="List item of 1"
    onClick={[Function]}
    test-id="List item of 1"
  >
    <img
      alt="img of 1"
      height="100%"
      src="https://images.amcnetworks.com/amc.com/wp-content/uploads/2015/04/cast_bb_700x1000_walter-white-lg.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Walter White
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 2"
    onClick={[Function]}
    test-id="List item of 2"
  >
    <img
      alt="img of 2"
      height="100%"
      src="https://upload.wikimedia.org/wikipedia/en/thumb/f/f2/Jesse_Pinkman2.jpg/220px-Jesse_Pinkman2.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Jesse Pinkman
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 3"
    onClick={[Function]}
    test-id="List item of 3"
  >
    <img
      alt="img of 3"
      height="100%"
      src="https://s-i.huffpost.com/gen/1317262/images/o-ANNA-GUNN-facebook.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Skyler White
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 4"
    onClick={[Function]}
    test-id="List item of 4"
  >
    <img
      alt="img of 4"
      height="100%"
      src="https://media1.popsugar-assets.com/files/thumbor/WeLUSvbAMS_GL4iELYAUzu7Bpv0/fit-in/1024x1024/filters:format_auto-!!-:strip_icc-!!-/2018/01/12/910/n/1922283/fb758e62b5daf3c9_TCDBRBA_EC011/i/RJ-Mitte-Walter-White-Jr.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Walter White Jr.
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 5"
    onClick={[Function]}
    test-id="List item of 5"
  >
    <img
      alt="img of 5"
      height="100%"
      src="https://upload.wikimedia.org/wikipedia/en/thumb/c/c1/Hank_Schrader2.jpg/220px-Hank_Schrader2.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Henry Schrader
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 6"
    onClick={[Function]}
    test-id="List item of 6"
  >
    <img
      alt="img of 6"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/1/10/Season_2_-_Marie.jpg/revision/latest?cb=20120617211645"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Marie Schrader
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 7"
    onClick={[Function]}
    test-id="List item of 7"
  >
    <img
      alt="img of 7"
      height="100%"
      src="https://images.amcnetworks.com/amc.com/wp-content/uploads/2015/04/cast_bb_700x1000_mike-ehrmantraut-lg.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Mike Ehrmantraut
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 8"
    onClick={[Function]}
    test-id="List item of 8"
  >
    <img
      alt="img of 8"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/1/16/Saul_Goodman.jpg/revision/latest?cb=20120704065846"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Saul Goodman
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 9"
    onClick={[Function]}
    test-id="List item of 9"
  >
    <img
      alt="img of 9"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/1/1f/BCS_S4_Gustavo_Fring.jpg/revision/latest?cb=20180824195925"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Gustavo Fring
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 10"
    onClick={[Function]}
    test-id="List item of 10"
  >
    <img
      alt="img of 10"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/b/b4/Hector_BCS.jpg/revision/latest?cb=20170810091606"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Hector Salamanca
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 11"
    onClick={[Function]}
    test-id="List item of 11"
  >
    <img
      alt="img of 11"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/e/e7/Krazy8.png/revision/latest?cb=20130208041554"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Domingo Molina
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 12"
    onClick={[Function]}
    test-id="List item of 12"
  >
    <img
      alt="img of 12"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/a/a7/Tuco_BCS.jpg/revision/latest?cb=20170810082445"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Tuco Salamanca
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 13"
    onClick={[Function]}
    test-id="List item of 13"
  >
    <img
      alt="img of 13"
      height="100%"
      src="https://images.amcnetworks.com/amc.com/wp-content/uploads/2015/04/cast_bb_700x1000_the-cousins-lg.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Marco & Leonel Salamanca
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 14"
    onClick={[Function]}
    test-id="List item of 14"
  >
    <img
      alt="img of 14"
      height="100%"
      src="https://media1.popsugar-assets.com/files/thumbor/wERDST0TUb-iHCSb2r5ZpsvaZLo/fit-in/1024x1024/filters:format_auto-!!-:strip_icc-!!-/2013/07/17/675/n/1922283/fae2f583f04bb80f_Laura-Fraser-is-back-as-Lydia-Rodarte-Quayle_gallery_primary/i/Laura-Fraser-Lydia-Rodarte-Quayle.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Lydia Rodarte-Quayle
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 15"
    onClick={[Function]}
    test-id="List item of 15"
  >
    <img
      alt="img of 15"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/9/95/Todd_brba5b.png/revision/latest?cb=20130717134303"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Todd Alquist
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 16"
    onClick={[Function]}
    test-id="List item of 16"
  >
    <img
      alt="img of 16"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/b/b4/Jane.jpg/revision/latest?cb=20090621233653"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Jane Margolis
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 17"
    onClick={[Function]}
    test-id="List item of 17"
  >
    <img
      alt="img of 17"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breaking-bad-tv/images/c/ce/Sp.png/revision/latest?cb=20121016143623"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Skinny Pete
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 18"
    onClick={[Function]}
    test-id="List item of 18"
  >
    <img
      alt="img of 18"
      height="100%"
      src="https://images.amcnetworks.com/amc.com/wp-content/uploads/2015/04/cast_bb_700x1000_badger-lg.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Brandon Mayhew
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 19"
    onClick={[Function]}
    test-id="List item of 19"
  >
    <img
      alt="img of 19"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/c/c1/4x11_-_Huell.png/revision/latest?cb=20130913100459&path-prefix=es"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Huell Babineaux
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 20"
    onClick={[Function]}
    test-id="List item of 20"
  >
    <img
      alt="img of 20"
      height="100%"
      src="https://images.amcnetworks.com/amc.com/wp-content/uploads/2015/04/cast_bb_700x1000_steven-gomez-lg.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Steven Gomez
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 21"
    onClick={[Function]}
    test-id="List item of 21"
  >
    <img
      alt="img of 21"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/b/bd/Cast_bb_700x1000_todd-beneke-lg.jpg/revision/latest?cb=20170723165057"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Theodore Beneke
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 22"
    onClick={[Function]}
    test-id="List item of 22"
  >
    <img
      alt="img of 22"
      height="100%"
      src="https://images.amcnetworks.com/amc.com/wp-content/uploads/2015/04/cast_bb_700x1000_gale-boetticher-lg.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Gale Boetticher
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 23"
    onClick={[Function]}
    test-id="List item of 23"
  >
    <img
      alt="img of 23"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/5/51/Andrea_-_To%27hajilee.png/revision/latest?cb=20131025094457"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Andrea Cantillo
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 24"
    onClick={[Function]}
    test-id="List item of 24"
  >
    <img
      alt="img of 24"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/9/97/Brock.png/revision/latest?cb=20170725193144"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Brock Cantillo
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 25"
    onClick={[Function]}
    test-id="List item of 25"
  >
    <img
      alt="img of 25"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/d/de/Carmen.JPG/revision/latest?cb=20100410125233"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Carmen Molina
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 26"
    onClick={[Function]}
    test-id="List item of 26"
  >
    <img
      alt="img of 26"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/5/5e/Gretchen_Schwartz.png/revision/latest?cb=20131005103735&path-prefix=es"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Gretchen Schwartz
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 27"
    onClick={[Function]}
    test-id="List item of 27"
  >
    <img
      alt="img of 27"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/8/8a/Elliott_Schwartz.png/revision/latest?cb=20131005103743&path-prefix=es"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Elliot Schwartz
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 28"
    onClick={[Function]}
    test-id="List item of 28"
  >
    <img
      alt="img of 28"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/d/d6/Gonzo.JPG/revision/latest?cb=20100415212944"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Gonzo
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 29"
    onClick={[Function]}
    test-id="List item of 29"
  >
    <img
      alt="img of 29"
      height="100%"
      src="https://res.cloudinary.com/dwvrok1le/image/upload/v1540314304/c8acek3pimb0hb4efrvf.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Christian Ortgea
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 30"
    onClick={[Function]}
    test-id="List item of 30"
  >
    <img
      alt="img of 30"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/a/a2/Tess_Harper.jpg/revision/latest?cb=20120923235754"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Mrs. Pinkman
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 31"
    onClick={[Function]}
    test-id="List item of 31"
  >
    <img
      alt="img of 31"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/c/c9/AdamPinkman.png/revision/latest?cb=20141117180836&path-prefix=es"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Adam Pinkman
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 32"
    onClick={[Function]}
    test-id="List item of 32"
  >
    <img
      alt="img of 32"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/a/a4/Jake.jpg/revision/latest?cb=20121214201656&path-prefix=de"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Jake Pinkman
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 33"
    onClick={[Function]}
    test-id="List item of 33"
  >
    <img
      alt="img of 33"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/c/ca/No_Doze_2.png/revision/latest?cb=20131014073210"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        No-Doze
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 34"
    onClick={[Function]}
    test-id="List item of 34"
  >
    <img
      alt="img of 34"
      height="100%"
      src="https://vignette.wikia.nocookie.net/trbreakingbad/images/9/9c/Emilio.jpeg/revision/latest?cb=20161029192022"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Emilio Koyama
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 35"
    onClick={[Function]}
    test-id="List item of 35"
  >
    <img
      alt="img of 35"
      height="100%"
      src="https://res.cloudinary.com/dwvrok1le/image/upload/v1540314509/lj5qlbe6xvmmpxnspz6y.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Dr. Delcavoli
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 36"
    onClick={[Function]}
    test-id="List item of 36"
  >
    <img
      alt="img of 36"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/c/cc/Wendy.jpg/revision/latest?cb=20170721162715"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Wendy S.
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 37"
    onClick={[Function]}
    test-id="List item of 37"
  >
    <img
      alt="img of 37"
      height="100%"
      src="https://i.pinimg.com/originals/d5/c0/34/d5c0345ae70fbdbaa33b7537d685da54.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Bogdan Wolynetz
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 38"
    onClick={[Function]}
    test-id="List item of 38"
  >
    <img
      alt="img of 38"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/8/8c/Ken.png/revision/latest?cb=20170721163649"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Ken
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 39"
    onClick={[Function]}
    test-id="List item of 39"
  >
    <img
      alt="img of 39"
      height="100%"
      src="https://pmctvline2.files.wordpress.com/2013/09/breaking-bad-elanor-anne-wenrich-325.jpg?w=325&h=240"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Holly White
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 40"
    onClick={[Function]}
    test-id="List item of 40"
  >
    <img
      alt="img of 40"
      height="100%"
      src="https://m.media-amazon.com/images/M/MV5BMTkwMTkxNjUzM15BMl5BanBnXkFtZTgwMTg5MTczMTE@._V1_UY317_CR175,0,214,317_AL_.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        George Merkert
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 41"
    onClick={[Function]}
    test-id="List item of 41"
  >
    <img
      alt="img of 41"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/1/12/Donald_Margolis.png/revision/latest?cb=20120802212034"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Donald Margolis
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 42"
    onClick={[Function]}
    test-id="List item of 42"
  >
    <img
      alt="img of 42"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/0/0f/Clovis.JPG/revision/latest?cb=20100415221013"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Clovis
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 43"
    onClick={[Function]}
    test-id="List item of 43"
  >
    <img
      alt="img of 43"
      height="100%"
      src="https://res.cloudinary.com/dwvrok1le/image/upload/v1539360786/o7thsjdorobfbetqaefj.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        SAC Ramey
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 44"
    onClick={[Function]}
    test-id="List item of 44"
  >
    <img
      alt="img of 44"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/4/48/Victor2.png/revision/latest?cb=20131009225027"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Victor
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 45"
    onClick={[Function]}
    test-id="List item of 45"
  >
    <img
      alt="img of 45"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/e/ec/Tomas.png/revision/latest?cb=20130131043014"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Tomás Cantillo
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 46"
    onClick={[Function]}
    test-id="List item of 46"
  >
    <img
      alt="img of 46"
      height="100%"
      src="https://res.cloudinary.com/dwvrok1le/image/upload/v1539362182/o9mstjonfn6dsdi3uij6.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Francesca Liddy
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 47"
    onClick={[Function]}
    test-id="List item of 47"
  >
    <img
      alt="img of 47"
      height="100%"
      src="https://res.cloudinary.com/dwvrok1le/image/upload/v1539362375/z63youp272k5mbqbuqj9.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Cynthia
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 48"
    onClick={[Function]}
    test-id="List item of 48"
  >
    <img
      alt="img of 48"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/5/58/BBWA_Tortuga.jpg/revision/latest?cb=20131101172007"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Tortuga
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 49"
    onClick={[Function]}
    test-id="List item of 49"
  >
    <img
      alt="img of 49"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/8/8d/2x02_-_Tim_Roberts.jpg/revision/latest?cb=20110927053954"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Tim Roberts
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 50"
    onClick={[Function]}
    test-id="List item of 50"
  >
    <img
      alt="img of 50"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/3/38/Bolsa.png/revision/latest?cb=20180825204033"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Juan Bolsa
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 51"
    onClick={[Function]}
    test-id="List item of 51"
  >
    <img
      alt="img of 51"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/8/81/Group_Leader_Brown_Shirt_-_Kafkaesque.png/revision/latest?cb=20131025095333"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Group Leader
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 52"
    onClick={[Function]}
    test-id="List item of 52"
  >
    <img
      alt="img of 52"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/c/cc/Kaylee.png/revision/latest?cb=20170810082843"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Kaylee Ehrmantraut
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 53"
    onClick={[Function]}
    test-id="List item of 53"
  >
    <img
      alt="img of 53"
      height="100%"
      src="https://res.cloudinary.com/dwvrok1le/image/upload/v1539373707/bbb6muek0tcmbc3g2sog.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Pamela
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 54"
    onClick={[Function]}
    test-id="List item of 54"
  >
    <img
      alt="img of 54"
      height="100%"
      src="https://res.cloudinary.com/dwvrok1le/image/upload/v1539373871/ssk1t4hvo9tmhvdt5hcn.jpg"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Duane Chow
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 55"
    onClick={[Function]}
    test-id="List item of 55"
  >
    <img
      alt="img of 55"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/b/b3/StaceyEhrmantraut.png/revision/latest?cb=20150310150049"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Stacey Ehrmantraut
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 56"
    onClick={[Function]}
    test-id="List item of 56"
  >
    <img
      alt="img of 56"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/f/f3/Officer_Saxton_-_I.F.T..png/revision/latest?cb=20131025090606"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Officer Saxton
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 57"
    onClick={[Function]}
    test-id="List item of 57"
  >
    <img
      alt="img of 57"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/c/ce/Jack5x13.png/revision/latest?cb=20130912225922"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Jack Welker
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 112"
    onClick={[Function]}
    test-id="List item of 112"
  >
    <img
      alt="img of 112"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/f/f7/BCS_S4_Kim_Wexler.jpg/revision/latest?cb=20180824195845"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Kimberly Wexler
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 113"
    onClick={[Function]}
    test-id="List item of 113"
  >
    <img
      alt="img of 113"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/5/57/BCS_S4_Howard_Hamlin.jpg/revision/latest?cb=20180824195855"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Howard Hamlin
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 114"
    onClick={[Function]}
    test-id="List item of 114"
  >
    <img
      alt="img of 114"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/3/3e/BCS_S3_ChuckMcGill.jpg/revision/latest?cb=20170327185308"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Charles McGill
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 115"
    onClick={[Function]}
    test-id="List item of 115"
  >
    <img
      alt="img of 115"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/b/b1/BCS_S4_Nacho_Varga.jpg/revision/latest?cb=20180824195910"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Ignacio Varga
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 116"
    onClick={[Function]}
    test-id="List item of 116"
  >
    <img
      alt="img of 116"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/8/85/LaloProfileBCS.png/revision/latest?cb=20180925050152"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Eduardo Salamanca
      </h1>
    </div>
  </styled.li>
  <styled.li
    key="List item of 117"
    onClick={[Function]}
    test-id="List item of 117"
  >
    <img
      alt="img of 117"
      height="100%"
      src="https://vignette.wikia.nocookie.net/breakingbad/images/b/b3/StaceyEhrmantraut.png/revision/latest?cb=20150310150049"
      width="100%"
    />
    <div
      className="description"
    >
      <h1>
        Stacey Ehrmantraut
      </h1>
    </div>
  </styled.li>
</styled.ul>
`;
